src/libs/ui/searchsidebar.cpp

```diff
@@ -16,6 +16,7 @@
 #include <registry/listmodel.h>
 #include <registry/searchmodel.h>
 #include <registry/searchquery.h>
+#include <ui/mainwindow.h>
```

Comment:
This should be avoided. I'd rather have side bar emit signals that can be connected to in the main window.